packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/CameraAndroidCameraxPlugin.java

```diff
@@ -66,7 +63,11 @@ public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {
 
   @Override
   public void onAttachedToActivity(@NonNull ActivityPluginBinding activityPluginBinding) {
-    updateContext(activityPluginBinding.getActivity());
+    CameraAndroidCameraxPlugin plugin = new CameraAndroidCameraxPlugin();
```

Comment:
I think I missed this in the last PR. But I don't think it is necessary to create a new `CameraAndroidCameraxPlugin` instance. You should be able to just call `setUp` on the current instance, right?